ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -272,6 +272,12 @@
 {% include 'eos/ip-http-client-source-interfaces.j2' %}
 {# ip ssh client source interfaces #}
 {% include 'eos/ip-ssh-client-source-interfaces.j2' %}
+{# ip ftp client source interfaces #}
```

Comment:
please verify on EOS that the order of commands are correct. Example try to add to a large config and see where it renders and move the includes to the correct place.

Reply:
I have updated the sequence for ftp/tftp/telnet in latest commit

Reply:
I also meant in relation to other configuration bits. Not only between these.

Reply:
btw it is interesting that these client source interfaces are all in a single block in EOS with no `!` between them. I expect HTTP client is also in that group if you try. We should put them together in a single template and also a single table in the docs. Would you be up for doing that in this PR?

Reply:
Sure, I will update this.

Reply:
yes http is also in the same group

Reply:
I have updated the sequence as per the output I see in EOS , some of the commands were not supported so kept them at the same place as earlier.
Please let me know for any changes on sequence